### PR TITLE
feat(language): Belarusian Lacinka

### PR DESCRIPTION
**Pull request by Dmskrzh**
Added 2 new languages "belarusian_lacinka" and "belarusian_lacinka_1k"

### CODE REVIEW

File: frontend/static/languages/belarusian_lacinka.json [hunk 1]
```diff
@@ -0,0 +1,206 @@
+{
+  "name": "belarusian_lacinka",
+  "noLazyMode": true,
```
Comment (Miodec) [inline frontend/static/languages/belarusian_lacinka.json:3]: Why is no lazy mode set to true? Im pretty sure lazy mode will work for these words

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/languages/belarusian_lacinka.json (hunk 1 above)
Comment (Dmskrzh) [reply frontend/static/languages/belarusian_lacinka.json:3]: Yes, I agree

**Review by Dmskrzh — COMMENTED** [review]
Yes, I agree

**Comment by Dmskrzh** [comment]
How can I aproove your requested changes?

**Comment by Miodec** [comment]
> How can I aproove your requested changes?

You need to make the change yourself then request a new review

**Comment by Dmskrzh** [comment]
I made the same changes to two files and one passed the checks and the other didn't, strange

**Review by Dmskrzh — COMMENTED** [review]
Changed noLazyMode to false

**Review by Dmskrzh — COMMENTED** [review]
Changed noLazyMode to false

**Comment by Dmskrzh** [comment]
I think I made all what you wanted

**Review by Miodec — APPROVED** [review]

**Pull request merged**